src/carbonio-files-ui-common/views/components/SecondaryBarItem.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { screen } from '@testing-library/react';
+
+import { SecondaryBarItem } from './SecondaryBarItem';
+import { setup } from '../../utils/testUtils';
+
+/*
+ * SPDX-FileCopyrightText: 2023 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+describe('SecondaryBarItem', () => {
+	test('should render an upload badge with a fraction if expanded is true', () => {
+		const item = {
+			id: '1',
+			completeTotalBadgeCounter: '1/1'
+		};
+		setup(<SecondaryBarItem item={item} expanded />);
+		expect(screen.getByText(/1\/1/i)).toBeInTheDocument();
```

Comment:
https://github.com/testing-library/jest-dom/pull/339/files#:~:text=it%20is%20present%20in%20the%20document

Reply:
tobevisible implies tobeinthedoc